tutorials/sample_vqe_program/qiskit_runtime_vqe_program.ipynb

```diff
@@ -0,0 +1,1394 @@
+      "  Max execution time: 100000\n",
+      "  Input parameters:\n",
+      "    - hamiltonian:\n",
+      "      Description: Hamiltonian whose ground state we want to find.\n",
+      "      Type: list\n",
+      "      Required: True\n",
+      "    - ansatz:\n",
+      "      Description: Name of ansatz quantum circuit to use, default='EfficientSU2'\n",
+      "      Type: str\n",
+      "      Required: False\n",
+      "    - ansatz_config:\n",
+      "      Description: Configuration parameters for the ansatz circuit.\n",
+      "      Type: dict\n",
+      "      Required: False\n",
+      "    - x0:\n",
+      "      Description: Initial vector of parameters.\n",
+      "      Type: ndarray\n",
+      "      Required: False\n",
+      "    - optimizer:\n",
+      "      Description: Classical optimizer to use, default='SPSA'.\n",
+      "      Type: str\n",
+      "      Required: False\n",
+      "    - optimizer_config:\n",
+      "      Description: Configuration parameters for the optimizer.\n",
+      "      Type: dict\n",
+      "      Required: False\n",
+      "    - shots:\n",
+      "      Description: Number of shots to take per circuit.\n",
+      "      Type: int\n",
+      "      Required: False\n",
+      "    - use_measurement_mitigation:\n",
+      "      Description: Use measurement mitigation, default=False.\n",
+      "      Type: bool\n",
+      "      Required: False\n",
+      "  Interim results:\n",
+      "    - params:\n",
+      "      Description: Parameter vector at current optimization step\n",
+      "      Type: ndarray\n",
+      "  Returns:\n",
+      "    - result:\n",
+      "      Description: Final result in SciPy optimizer format.\n",
+      "      Type: OptimizeResult\n"
+     ]
+    }
+   ],
+   "source": [
+    "prog = provider.runtime.program(program_id)\n",
+    "print(prog)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bb2627e9",
+   "metadata": {},
+   "source": [
+    "### Deleting a program\n",
+    "\n",
+    "If you make a mistake and need to delete and/or re-upload the program you can run the following, passing the `program_id`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "a52bdcdf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#provider.runtime.delete_program(program_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b521258",
+   "metadata": {},
+   "source": [
+    "## Running the program\n",
+    "\n",
+    "### Specify parameters\n",
+    "\n",
+    "To run the program we need to specify the `options` that are used in the runtime environemnt (not the program variables).  At present, only the `backend_name` is required."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "10629ac7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "backend = provider.backend.ibmq_qasm_simulator\n",
+    "options = {'backend_name': backend.name()}"
```

Comment:
Jay has been pushing to do just
```
options = {'backend_name': 'ibmq_qasm_simulator'}
```
instead of initializing a separate `backend` object.